services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/StaticAttributeTags.java

```diff
@@ -117,6 +118,25 @@ public static StaticAttributeTag updateBehavior(UpdateBehavior updateBehavior) {
         return UpdateBehaviorTag.fromUpdateBehavior(updateBehavior);
     }
 
+    /**
+     * Used to explicitly designate an attribute to be an auto-generated counter updated unconditionally in DynamoDB.
+     * By supplying a negative integer delta value, the attribute works as a decreasing counter.
+     *
+     * @param delta The value to increment (positive) or decrement (negative) the counter with for each update.
+     * @param startValue The starting value of the counter.
+     */
+    public static StaticAttributeTag atomicCounter(long delta, long startValue) {
```

Comment:
Do you expect that there will be other parameters in the future?

Reply:
No, I cannot think of any